Heroes Handbook, Main File

```diff
@@ -979,6 +1025,52 @@ The worgen curse has transformed your body into that of a worgen, giving you the
 
 \pagebreakNum
 
+# Horde Class Suggestions
+
+The following are a set of class suggestions, pulled from World of Wacraft, as the classes the Horde races will more than likely be. It is suggested that you pick from among your race's suggested classes to keep with the thematic elements of the setting. That said, heroes are meant to break molds and be the exception to rules, so feel free to pick a class that is not typically used by your race. If you do pick outside your racial classes though, work with your Dungeon Master to create a backstory for why you are a member of this class. 
+
+You will likely notice that there are two class suggestion charts. One for Pre-Cataclysm, and one for Post-Cataclysm. This was done because the Cataclysm is a major turning point in the history of Azeroth, and the races of the Horde began learning new abilities and sources of power, in order to fight back against the threats that assail their world. Prior to the Cataclysm, the Horde races were not as open to sharing knowledge of and learning about new classes and tended to stay within the traditional classes of their race.
+
+<div class='wide'>
+
+## Pre-Cataclysm
+
+| Race | Class Suggestions |
+|:----|:-------------|
+| Orc (All Tribes) | Hunter, Rogue, Shaman, Warlock, Warrior, Death Knight |
+| Forsaken (Elf and Human) | Mage, Priest, Rogue, Warlock, Warrior, Death Knight |
+| Mulgore Tauren | Druid, Hunter, Shaman, Warrior, Death Knight |
+| Winter Tauren (Taunka) | Druid, Hunter, Shaman, Warrior |
+| Forest Troll | Hunter, Mage, Priest, Rogue, Shaman, Warlock, Warrior |
+| Frost Troll | Hunter, Priest, Warrior |
+| Jungle Troll | Hunter, Mage, Priest, Rogue, Shaman, Warrior, Death Knight | 
+| Blood Elf (Sin'dorei) | Hunter, Mage, Paladin, Priest, Rogue, Warlock, Death Knight |
+
+</div>
+
+<div class='wide'>
+
+## Post-Cataclysm
+
+| Race | Class Suggestions |
+|:----|:-------------|
+| Orc (All Tribes) | Hunter, Mage, Rogue, Shaman, Warlock, Warrior, Death Knight, Monk |
+| Forsaken (Elf and Human) | Hunter, Mage, Priest, Rogue, Warlock, Warrior, Death Knight, Monk |
```

Comment:
Maybe add Demon Hunter to forsakens as well, seeing as they are elves and humans, also evens it out with both factions having one 'none' elf race able of being a demon hunter

```suggestion
| Forsaken (Elf and Human) | Hunter, Mage, Monk, Priest, Rogue, Warlock, Warrior, Death Knight |
```